package.json

```diff
@@ -28,6 +28,10 @@
 	},
 	"dependencies": {
 		"@automattic/color-studio": "^2.5.0",
+		"@wordpress/components": "^19.5.0",
```

Comment:
These deps should be available already from [WooCommerceDependencyExtractionWebpackPlugin](https://github.com/Automattic/wc-calypso-bridge/blob/master/webpack.config.js#L11)

Reply:
I think we should still explicitly declare them as dependencies though? But yes we should set up the dep extraction properly so that it isn't re-included in the bundle (not sure if it is).

Reply:
I think we'll need the dependencies declared or dev tooling, although admittedly things like Jest are not established right now.

Reply:
> we should set up the dep extraction properly so that it isn't re-included in the bundle

That's a great idea, and it should also help with the JS linters!

> things like Jest are not established right now.

FYI, If we have a robust way of testing JS modules, we could definitely add it as part of the new GitHub Actions introduced in #926.